jupyter-on-gke/README.md

```diff
@@ -98,11 +105,18 @@ We allow user to set their own domains, in the `variables.tf` file. Since we are
 
 ### If Auth is disabled
 
+For demo purposes, this disabling auth creates a public IP for the Jupyter notebook with basic dummy authentication. To secure your cluster, it is _strongly recommended_ to enable authenticator and use IAP.
```

Comment:
I think we should instead rename the "Authentication (Currently Enabled by Default)" section to "Securing your Jupyterhub Endpoint", and move this sentence about strongly recommended enabling auth to that section. It doesn't belong in usage.

Reply:
I see, done!